repair/row_level.cc

```diff
@@ -2235,6 +2250,28 @@ future<> repair_uninit_messaging_service_handler() {
     });
 }
 
+class repair_meta_tracker {
+    boost::intrusive::list<repair_meta,
+        boost::intrusive::member_hook<repair_meta, repair_meta::tracker_link_type, &repair_meta::_tracker_link>,
+        boost::intrusive::constant_time_size<false>> _repair_metas;
+public:
+    void add(repair_meta& rm) {
+        _repair_metas.push_back(rm);
+    }
+};
+
+namespace debug {
+    extern thread_local repair_meta_tracker repair_meta_for_masters;
+    extern thread_local repair_meta_tracker repair_meta_for_followers;
+}
```

Comment:
Why do we need separate forward declaration and defninition later? It's in the same file, can't we just define here and be done with it?

Reply:
Nice! Yes, we can do that. I was fooled by class dependency thing.